packages/cli/internal/pkg/cli/workflow/manager.go

```diff
@@ -534,6 +576,14 @@ func (m *Manager) cleanUpAttachments() {
 			log.Warn().Msgf("Failed to clean up temporary file '%s': %s", attachment, err)
 		}
 	}
+	log.Debug().Msgf("removing extra directory")
+	err := removeAll("extra")
+	if err != nil {
+		log.Warn().Msgf("Failed to remove extra directory")
+		m.err = err
+		return
+	}
```

Comment:
I would refactor this into a separated method such as `removeTempManifast` to increase readability.